src/main/reseal.c

```diff
@@ -397,8 +474,21 @@ int main(int argc, char **argv)
     return 1;
```

Comment:
I think the previous two lines (472-473) may no longer be needed (assuming other suggestions are implemented)

Remove `free(output)` - replaced by `free(seal_output)` in the previous suggestion.

Remove `free(pcrs)` - freed earlier in a suggestion above.